shell/browser/native_window_mac.mm

```diff
@@ -472,6 +472,7 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
   }
 
   [window_ performClose:nil];
+  window_ = nil;
```

Comment:
The window is not closed after `performClose` is called, Electron will prevent the close in `windowShouldClose` and then ask the WebContents to close, and the window will then be closed in `WebContentsDestroyed`.

This is why macOS CI is crashing.